### PR TITLE
[PCADMIN-5567] - fixing issue with localization on arrays

### PR DESCRIPTION
**Pull request by jordanstith15**

### CODE REVIEW

File: src/global/i18n.ts [hunk 1]
```diff
@@ -64,6 +66,13 @@ export default (function() {
             return untranslated;
           }
         }
+        if (options) {
+          for ( const key of Object.keys(options)){
+            console.log(key);
```
Comment (pmoaddel) [inline src/global/i18n.ts]: accidentally left this console statements in?

File: src/components/json-schema-designer/schema.ts [hunk 2]
```diff
@@ -541,7 +543,7 @@ export class SchemaArray extends SchemaRoot implements ISchemaItem {
   removeChild(id: string): void {
     let itemRemoved: boolean = false;
     this.items = this.items.filter((item) => {
-        if (item._id = id) {
+        if (item._id == id) {
```
Comment (pmoaddel) [inline src/components/json-schema-designer/schema.ts:546]: yikes good catch

File: src/global/i18n.ts (hunk 1 above)
Comment (brian-slate) [reply src/global/i18n.ts]: Remove or convert to debugs if this is useful.

**Review by brian-slate — APPROVED** [review]
Looks great other than those console statements.

File: src/global/i18n.ts (hunk 1 above)
Comment (jordanstith15) [reply src/global/i18n.ts]: yeah I did. They've been removed now

**Review by pmoaddel — APPROVED** [review]

**Pull request merged**